jsonrpc/helper.go

```diff
@@ -30,7 +30,13 @@ type latestHeaderGetter interface {
 // GetNumericBlockNumber returns block number based on current state or specified number
 func GetNumericBlockNumber(number BlockNumber, store latestHeaderGetter) (uint64, error) {
 	switch number {
-	case LatestBlockNumber, PendingBlockNumber:
+	case SafeBlockNumber:
+		fallthrough
+	case FinalizedBlockNumber:
+		fallthrough
+	case LatestBlockNumber:
+		fallthrough
+	case PendingBlockNumber:
 		latest := store.Header()
```

Comment:
Should we rename latest variable? ErrLatestNotFound?

Reply:
The `safe`, `finalized`, `latest` and `pending` are all behaving the same way and are resolving the latest block. So IMO it is correct to keep it as is (since because of the instant finality and no reorgs, we do not have finalized and safe as the defacto values).